packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -11,7 +18,7 @@ export function truncate(fileString: string, truncateMarker: RegExp | string) {
 }
 
 // YYYY-MM-DD-{name}.mdx?
-// prefer named capture, but old node version do not support
+// prefer named capture, but old Node version does not support.
 const FILENAME_PATTERN = /^(\d{4}-\d{1,2}-\d{1,2})-?(.*?).mdx?$/;
 
 function toUrl({date, link}: DateLink) {
```

Comment:
@endiliey why is there a hardcoded date 2019-01-01 here? On L53 as well o.o

Reply:
Not me who put that. Iirc its hardcoded that way by some prevs contributor who added cjk support.

He wanted to make it clear that its yyyy-mm-dd and then take the length lol.

Reply:
Maybe just substring(0,9) lol